app/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/nav_graph"
+    app:startDestination="@id/citySelectorFragment">
+
+    <fragment
+        android:id="@+id/citySelectorFragment"
+        android:name="com.weather.weathercheckerapp.fragment.CitySelectorFragment"
+        android:label="cityselector_container"
```

Comment:
This label is a bit strange. I think this should be the name of the fragment xml: "fragment_city_selector". In fact, as I see, if you don'T do that, the nav graph editor won't show the preview of that fragment.